skfp/preprocessing/filters/ghose.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional
+
+from rdkit.Chem import Mol
+from rdkit.Chem.Crippen import MolLogP, MolMR
+from rdkit.Chem.Descriptors import MolWt
+from rdkit.Chem.rdMolDescriptors import CalcNumAtoms
+
+from skfp.bases.base_filter import BaseFilter
+
+
+class GhoseFilter(BaseFilter):
+    """
+    Ghose rule.
+
+    Used to searching for drug-like  molecules [1]_.
+
+    Molecule must fulfill conditions:
+
+        - 160 <= molecular weight <= 400
+        - -0.4 <= logP <= 5.6
+        - 20 <= Natoms <=70
```

Comment:
`number of atoms`? Also, if you are only counting heavy atoms, you could write `heavy atom count (HAC)` here. And space is missing after second `<=`